pkg/webhook/admission/webhook.go

```diff
@@ -253,3 +253,21 @@ func StandaloneWebhook(hook *Webhook, opts StandaloneOptions) (http.Handler, err
 	}
 	return metrics.InstrumentedHook(opts.MetricsPath, hook), nil
 }
+
+// requestContextKey is how we find the admission.Request in a context.Context.
+type requestContextKey struct{}
```

Comment:
How about a const string like `"webhookRequest"`?

Reply:
I think the idea behind using an unexported struct is to only make it possible to retrieve it with the exported `RequestFromContext` func.

(I took this patter from how go-logr stores a Logger in a context)